mathics/builtin/atomic/symbols.py

```diff
@@ -400,6 +400,29 @@ def eval(self, symbol, evaluation):
         return get_symbol_values(symbol, "DownValues", "downvalues", evaluation)
 
 
+# In Mathematica 5, this appears under "Types of Values".
+class FormatValues(Builtin):
+    """
+    <url>:WMA link: https://reference.wolfram.com/language/ref/FormatValues.html</url>
```

Comment:
This is a nonexistent link. 

I am curious why there is interest in this right now.

Reply:
Because I was running some experiments which work on WMA, and not here.

Reply:
Ok. But please let's defer Boxing and Formatting work until after release. I am having problems with the amount of work needed to go over for the current release. Thanks.

Reply:
If you are looking for something to do that will assist with the next release remember when you wrote https://github.com/Mathics3/mathics-core/pull/1290#discussion_r1915855699 : 
> Mathics-Django uses Mathjax to render equations, so it is just a matter to call the render when the documentation is loaded. I will look how to do this later.

Reply:
It seems that this symbol remains undocumented since WMA v3...
 https://reference.wolfram.com/legacy/v3/MainBook/A.5.3.html

Reply:
Changing the link to that URL is fine. And adding this Builtin which still exists in WMA 13 is okay too. Bu all of this is a distraction from the next release. I'd appreciate it PRs could be more focused towards the next release.